pkg/cmd/server/origin/asset.go

```diff
@@ -164,7 +164,13 @@ func (c *AssetConfig) addHandlers(mux *http.ServeMux) error {
 	versions := sets.NewString()
 	versions.Insert(latest.Versions...)
 	versions.Insert(klatest.Versions...)
+	deadVersions := sets.NewString()
+	deadVersions.Insert(configapi.DeadKubernetesAPILevels...)
+	deadVersions.Insert(configapi.DeadOpenShiftAPILevels...)
 	for _, version := range versions.List() {
+		if deadVersions.Has(version) {
```

Comment:
also remove `MasterLegacyPrefix` from `WebConsoleConfig` and this line from the config template:
```
        "v1beta3": "{{ .MasterLegacyPrefix | js}}",
```